README.md

```diff
@@ -164,6 +165,77 @@ After refactoring your downstream model to select from the staging layer, your D
 <img width="500" alt="image" src="https://user-images.githubusercontent.com/73915542/165100261-cfb7197e-0f39-4ed7-9373-ab4b6e1a4963.png">
 </details>
 
+#### Hard Coded References
+`fct_hard_coded_references` ([source](models/marts/dag/fct_hard_coded_references.sql)) shows each instance where a model contains hard coded references. 
+
+<details>
+<summary><b>Example</b></summary>
+
+`fct_orders` uses hard coded direct relation references.
+
+```
+# fct_orders.sql
+
+with orders as (
+    select * from my_db.my_schema.orders
+),
+customers as (
+    select * from my_schema.customers
+)
+select
+    orders.order_id,
+    customers.name
+from orders
+left join customers on
+	orders.customer_id = customers.id
+```
+
+</details>
+
+<details>
+<summary><b>Reason to Flag</b></summary>
+
+Always use the `ref` function when selecting from another model and the `source` function when selecting from raw data, rather than using the direct relation reference (e.g. `my_schema.my_table`). Direct relation references are determined via regex mapping [here](macros/find_all_hard_coded_references.sql). 
+
+The `ref` and `source` functions are part of what makes dbt so powerful! Using these functions allows dbt to infer dependencies (and check that you haven't created any circular dependencies), properly generate your DAG, and ensure that models are built in the correct order. This also ensures that your current model selects from upstream tables and views in the same environment that you're working in.
+
+</details>
+
+<details>
+<summary><b>Exceptions</b></summary>
+
+TO DO: Are there any exceptions anyone can think of?? I think there might be some packages that select from a variable...
```

Comment:
Should we remove the TODO?

Reply:
yes, but this is an open question - can you think of any exceptions?

Reply:
No, and if there is any, then people will just use the seed file I guess